src/tests/quotes.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render, act } from '@testing-library/react';
+import Quotes from '../components/quotes';
+
+describe('Quotes', () => {
+  it('should render loading message while quotes are being fetched', () => {
+    const { container } = render(<Quotes />);
+    expect(container.firstChild).toMatchSnapshot();
+  });
+  it('should render failed message if quotes failed to fetch', async () => {
+    jest.spyOn(global, 'fetch').mockImplementation(() => {
+      Promise.reject(new Error('Failed to fetch quotes'));
+    });
+    await act(async () => {
+      render(<Quotes />);
+    });
+  });
```

Comment:
- [ ] You did a great job with most of the tests. This particular test is failing, I think one problem is that you don't have any `expect` clause for this particular test to confirm your expectation. Please make sure this test passes. 🤝️

![Screenshot from 2023-05-12 12-27-38](https://github.com/naanmohammed/Math-Magician/assets/17243325/33a9401a-eb16-47ad-935a-fc3ed1f0243d)